qiita_db/test/test_sql_connection.py

```diff
@@ -611,5 +612,13 @@ def test_index(self):
             trans.add(sql, args, many=True)
             self.assertEqual(trans.index, 7)
 
+    def test_get_transaction(self):
+        trans = Transaction("test_get_transaction")
+        obs = get_transaction(trans, "test")
+        self.assertEqual(obs, trans)
+
+        obs = get_transaction(None, "test")
+        self.assertEqual(obs._name, "test")
```

Comment:
Now, but another class may have that as a class attribute in the future. Better to be safe and check explicitly.